### DISCUSSION TITLE
Output is not standard - Raw JSON Output expected, but ```json ``` embedded received

### DISCUSSION THREAD

**Discussion by rocketkay** [crewAIInc/crewAI#1325] [General]
Hey guys,
i could really say thanks this this wonderful project. And i am in.

I wrote an agent that can optimize a product content. My expectation is a json output. 

``
Task( 
         description="",
        expected_output='valid parsable JSON format with {title:text, description:text, bullet_point:List[string] }. Nothing else ',
``

so 
60% is {}
40% is ```json {} ```

i know there are efforts in the direction to use chatGPT output variables. 

But what do you would use instead?

**Comment by R3xpook** [comment]
I saw that with training it output a correct JSON but....than i have problem with the training so i don't know it might work for you ?

**Reply by R3xpook** [reply]
Also making a list of thing that NOT TO DO like not using markdown and specify to not use caps and not use extra comments or characters outside the json structure
